typhon/objects/networking/endpoints.py

```diff
@@ -321,3 +346,12 @@ def makeTCP4ServerEndpoint(port):
     """
 
     return TCP4ServerEndpoint(unwrapInt(port))
+
+
+@runnable(RUN_1)
+def makeTCP6ServerEndpoint(port):
```

Comment:
I don't believe there are any tests for this. Though, it is now in the unsafeScope.